BE/PoroDev.Backend.sln

```diff
@@ -3,7 +3,7 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 17
 VisualStudioVersion = 17.2.32505.173
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Service.Access.Layer", "Service.Access.Layer\Service.Access.Layer.csproj", "{C8D3DFE9-E254-46FD-81EE-EDACC6C014D5}"
```

Comment:
When renaming a project from VS the folder name remains the same. Rename the folder manually.

Reply:
Done